src/App.js

```diff
@@ -1,19 +1,34 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route } from 'react-router-dom'
+import './components/common/Theme/Global'
+
+import Home from './components/Home'
+import Favorites from './components/Favorites'
+import Navbar from './components/common/Navbar';
+import Detail from './components/Deatail';
+
+
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
+
+
+      <div>
+        <header>
+          <Navbar />
+          
         </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+        <div>
+          <Route exact path="/" component={Home} /> 
+          <Route exact path="/favorites" component={Favorites} />
```

Comment:
exact prop is not needed since there is no conflict between "/favorites" and any other route, for exclusiveness behavior between routes in the same level like this, it is better to wrap them in a [Switch](https://reacttraining.com/react-router/web/api/Switch)